tests/operators/test_snowflake_decorator.py

```diff
@@ -73,15 +74,41 @@ class TestSnowflakeOperator(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.snowflake_table = test_utils.get_table_name(
+            "SNOWFLAKE_TRANSFORM_TEST_TABLE"
+        )
+        cls.snowflake_table_raw_sql = test_utils.get_table_name(
+            "SNOWFLAKE_TRANSFORM_RAW_SQL_TEST_TABLE"
+        )
```

Comment:
I had no idea we could define variables at a class level like this. Great find!

Reply:
Thanks 👍